cpp/scenario/gazebo/src/World.cpp

```diff
@@ -227,6 +228,15 @@ bool World::createECMResources()
             m_ecm, m_entity, std::chrono::steady_clock::duration::zero());
     }
 
+    // Print the active physics profile
+    const auto& physics = utils::getExistingComponentData< //
+        ignition::gazebo::components::Physics>(m_ecm, m_entity);
+    sDebug << "Initializing world '" << this->name()
```

Comment:
Did you intended to commit a debug print or it was a leftover?

Reply:
This is intended, the same code was printed in the `GazeboSimulator` before.